src/util/component-types.spec.js

```diff
@@ -230,7 +234,7 @@ describe('component-types', () => {
 			);
 		});
 
-		it('should find all elements of the same type from props and children', () => {
+		it('should find all elements of the same type from props and children', (done) => {
```

Comment:
yeah, I'm not too sure why it was failing after the upgrade, either.

This fixed it though.